README.md

```diff
@@ -10,27 +10,26 @@ sending an RPC request to a server receiving it.
 In this case, the same trace IDs are used, which means that both the client and
 server side of an operation end up in the same node in the trace tree.
 
-Here's an example flow, assuming an HTTP request carries the propagated trace:
+Here's an example flow using multiple headers, assuming an HTTP request carries the propagated trace:
```

Comment:
using multiple header encoding? (as you explain the types later in the HTTP encoding section)

Reply:
ack